### PR TITLE
Merge step 1 and 2 of crowdfunding form

### PR DESCRIPTION
**Pull request by pcP1r4t3**
This will resolve [meta#92](https://github.com/Coinsence/meta/issues/92)

### CODE REVIEW

File: views/funding/create.php [hunk 1]
```diff
@@ -17,22 +18,38 @@
 Select2BootstrapAsset::register($this);
 ?>
 
-<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset'), 'closable' => false]) ?>
+<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset and define exchange rate'), 'closable' => false]) ?>
```
Comment (craxrev) [inline views/funding/create.php]: To accomodate the new translation format coming in the next module version, please change all `XcoinModule.base` occurences in your modified files (only files commited in this pr) to `XcoinModule.${view_name}` considering the `view_name` as in `views/${view_name}/`, like in this line `XcoinModule.base` -> `XcoinModule.funding`

File: views/funding/create.php [hunk 2]
```diff
@@ -17,22 +18,38 @@
...
+            <?= $form->field($model, 'exchange_rate')->widget(AmountField::class, ['asset' => $myAsset])->label(Yii::t('XcoinModule.base', 'Provided asset')); ?>
+        </div>
+        <div class="col-md-2 text-center">
+            <i class="fa fa-exchange colorSuccess" style="font-size:28px;padding-top:24px" aria-hidden="true"></i>
+        </div>
+        <div class="col-md-5">
+            <?=
+            $form->field($model, 'asset_id')->widget(Select2::classname(), [
+                'data' => $assetList,
+                'options' => ['placeholder' => '- Select asset - ', 'value' => ($defaultAsset) ? $defaultAsset->id : []],
```
Comment (craxrev) [inline views/funding/create.php]: translate this: change to `['placeholder' => '- ' . Yii::t('XcoinModule.funding', 'Select asset') . ' - ', 'value' => ($defaultAsset) ? $defaultAsset->id : []],`

Comment (gdalyy) [reply views/funding/create.php]: I think we need to merge translation PR first , then handle the translation in this PR .

**Pull request merged**